homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -8,7 +8,7 @@
 import logging
 from typing import TYPE_CHECKING
 
-from homeassistant import config_entries, core
+from homeassistant import core
```

Comment:
I know this was already there - but you should avoid importing core as a module.
You should just import the object you need (`from homeassistant.core import HomeAssistant`)

Reply:
Will change in another PR

Reply:
Is already like that in the unit tests